docs/stencil-docs/storefront-customization/custom-templates.md

```diff
@@ -38,7 +38,7 @@ If you are migrating from BigCommerce's legacy Blueprint themes framework, pleas
 
 ## Authoring a custom template
 
-As a theme developer, you must first create the custom subdirectory in the `templates/pages` directory, and the four required subdirectories inside of it (brand, category, product, page). Creating these subdirectories will result in the following directory paths:
+As a theme developer, you must first create a custom page. Create the custom page by going to **Storefront** > **Web Pages** in the Control Panel. Then create the custom subdirectory in the `templates/pages` directory, and the four required subdirectories inside of it (brand, category, product, page). Creating these subdirectories will result in the following directory paths:
```

Comment:
Changed spelling per our style guide: [BigCommerce Glossary](https://intranet.bigcommerce.com/display/SST/BigCommerce+Glossary#BigCommerceGlossary-C)
```suggestion
As a theme developer, you must first create a custom page. Create the custom page by going to **Storefront** > **Web Pages** in the control panel. Then create the custom subdirectory in the `templates/pages` directory, and the four required subdirectories inside of it (brand, category, product, page). Creating these subdirectories will result in the following directory paths:
```

Reply:
Yes, you are right. Thank you for catching this.